### PR TITLE
[T4A3][W15-B3]Zachary Lim Wei Ling

### PR DESCRIPTION
**Pull request by zachylimwl**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Parent class of Address, Email and Phone
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:4]: the comment shouldn't talk about the child classes. 
header comments can include what it is, how it is used but shouldn't include where it should be used

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+	@Override
+	public String toString() {
+		return value;
+	}
+
+	@Override
+	public boolean equals(Object other) {
+		return other == this // short circuit if same object
+				|| (other instanceof Contact // instanceof handles nulls
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:23]: can you see why this is a problem? you can now potentially compare `Address` with `Email`, why?

**Comment by okkhoy** [comment]
- how about extending the `Address`, `Email` etc., using `Contact`?
  this seems incomplete.

**Comment by okkhoy** [comment]
@zachylimwl 
Some comments added. Please ack & close the PR after reading comments.

**Comment by zachylimwl** [comment]
@okkhoy because the instructions only asked to extract the commonalities... didn't ask to actually extend using Contact...

**Pull request closed without merging**